vignettes/Gating_1.Rmd

```diff
@@ -19,12 +19,15 @@ Required libraries
 ```{r load_library}
 library(flowWorkspace)
 library(ggcyto)
+library(CytoverseBioc2023)
+cache_workshop_data()
 ```
 
 Steps to create a `GatingSet`
 ```{r intro_gatingset, eval = T, echo=TRUE}
 # create a cytoset
-cs <- flowWorkspace::load_cytoset_from_fcs(path = "data/fcs_data/",pattern = "TNK-CR1")
+fcs_files <- get_workshop_data("fcs_data/")
```

Comment:
And get the actual local paths to the files with `get_workshop_data` then `$rpath`